projects/ad9361/src/main.c

```diff
@@ -160,7 +160,7 @@ struct axi_dac_init tx_dac_init = {
 struct axi_dmac_init rx_dmac_init = {
 	"rx_dmac",
 	CF_AD9361_RX_DMA_BASEADDR,
-#ifdef ADC_DMA_IRQ_EXAMPLE
```

Comment:
any specific reason for the rename? the previous versions looks more explicit for the user. Also, if you do this change please make sure that the .json file is updated accordingly.

Reply:
In the current implementation, the define is valid for both ADC and DAC, and not only for the first one. Therefore, removing ADC from the define makes the example selection clearer.

Reply:
got it, thanks.